### PR TITLE
Snackbar

### PR DESCRIPTION
**Pull request by Lokeshkhati**
image

Kindly review this component

### CODE REVIEW

File: css/components/input.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  
+  .required-field:after {
+    content: "*";
+    color: var(--danger-color);
+  }
+  
+  .form-field {
+    width: 90%;
+    display: block;
+    padding: 8px;
```
Comment (shubhamkumar648) [inline css/components/input.css]: use rem units

File: css/components/input.css [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  }
+  
+  .form-field:focus {
+    border: 2px solid var(--primary-color);
+  }
+  
+  .form-validation-msg {
+    margin-top: 0.25rem;
+    font-size: 0.875rem;
+    line-height: 21px;
```
Comment (shubhamkumar648) [inline css/components/input.css]: same here

**Review by shubhamkumar648 — COMMENTED** [review]
avoid using px value use rem for better web performance other than that looks good

File: css/components/input.css (hunk 2 above)
Comment (Lokeshkhati) [reply css/components/input.css]: Thanks for your suggestion

**Review by raman1999 — APPROVED** [review]
Everything looks good to me

File: css/components/input.css (hunk 1 above)
Comment (Lokeshkhati) [reply css/components/input.css]: Thanks for your suggestion.

**Pull request merged**